packages/modules/src/modules/banners/designableBanner/DesignableBanner.tsx

```diff
@@ -378,10 +379,14 @@ const DesignableBanner: ReactComponent<BannerRenderProps> = ({
 };
 
 const styles = {
-    outerContainer: (background: string, textColor: string = 'inherit') => css`
+    outerContainer: (
+        background: string,
+        limitHeight: boolean,
+        textColor: string = 'inherit',
+    ) => css`
         background: ${background};
         color: ${textColor};
-        max-height: 100vh;
+        ${limitHeight ? 'max-height: 70vh;' : ''}
```

Comment:
I've removed the default `100vh` because it gets overridden by DCR, where max-height of a parent is set